integrations/access/pagerduty/app.go

```diff
@@ -312,10 +343,26 @@ func (a *App) getOnCallServiceNames(req types.AccessRequest) ([]string, error) {
 	return common.GetServiceNamesFromAnnotations(req, annotationKey)
 }
 
+func (a *App) getMessageRecipient(ctx context.Context, req types.AccessRequest) (string, error) {
+	recipientSetService := a.accessMonitoringRules.RecipientsFromAccessMonitoringRules(ctx, req)
+	if recipientSetService.Len() > 1 {
+		return "", trace.BadParameter("more than one service provided as PagerDuty plugin recipient")
```

Comment:
Do we not support this? Can we just take the first one instead of hard-failing?

Reply:
Since these could be coming from separate rules with an undefined evaluation order taking the first would be pretty much just taking a random one which I doubt would be any users intent. I think failing to post anything would make more sense until we have the ability to set evaluation order for rules.